src/viem.ts

```diff
@@ -0,0 +1,686 @@
+import {Chain, mainnet} from "viem/chains";
+import {
```

Comment:
could we just do `import * as allViemChains` and iterate through that instead of specifying each one? Otherwise when new chains are added (e.g. etherlink mainnet), we'll have to manually update this import for each new chain

Reply:
We can't do `import * as allViemChains` directly because the file also exports types https://github.com/wevm/viem/blob/main/src/chains/index.ts#L316-L406

I think we can come around with a workaround though, I'll spend some time for this today

Reply:
done